front/app/components/EventAttendanceButton/ConfirmationModal.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { Box, Title, Text } from '@citizenlab/cl2-component-library';
+
+import EventSharingButtons from 'containers/EventsShowPage/components/EventSharingButtons';
+
+import { AddEventToCalendarButton } from 'components/AddEventToCalendarButton';
+import Modal from 'components/UI/Modal';
+
+import { EventModalConfetti } from './EventModalConfetti';
+import messages from './messages';
+
+import useAuthUser from 'api/me/useAuthUser';
+import { useIntl } from 'utils/cl-intl';
+import { IEventData } from 'api/events/types';
+import useLocalize from 'hooks/useLocalize';
+import { getEventDateString } from 'utils/dateUtils';
+
+interface Props {
+  opened: boolean;
+  event: IEventData;
+  onClose: () => void;
+}
+
+const ConfirmationModal = ({ opened, event, onClose }: Props) => {
```

Comment:
Just extracted this form the `EventAttendanceButton/index` to clean up the code a bit